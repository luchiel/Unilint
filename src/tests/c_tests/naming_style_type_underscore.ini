[naming_styles]
class_naming_style = underscore
