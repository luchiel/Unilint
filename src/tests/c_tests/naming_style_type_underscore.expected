12:8: name AbcAbc is not of style underscore
13:8: name abcAbc is not of style underscore
14:8: name ABC_ABC is not of style underscore
15:8: name Abc_abC is not of style underscore
26:9: name SabakA is not of style underscore
27:9: name sabakA is not of style underscore
28:9: name SABAKA_ is not of style underscore
29:9: name SaB_aKa is not of style underscore
